lib/core/elastic_connector_actions.rb

```diff
@@ -157,12 +157,18 @@ def update_connector_sync_now(connector_id, sync_now)
         )
       end
 
-      def update_connector_last_sync_status(connector_id, last_sync_status)
+      def update_connector_sync_start(connector_id)
         doc = connector_with_concurrency_control(connector_id)
 
+        body = {
+            last_sync_status: Connectors::SyncStatus::IN_PROGRESS,
+            last_sync_error: nil,
+            status: Connectors::ConnectorStatus::CONNECTED
```

Comment:
Why update status here?

Reply:
A new job gets claimed and starts execution. we should mark the job as `connected`, otherwise it may stay `error` until job completes.

Reply:
Ah I see, overlooked it.